scripts/variantstore/beta_docs/gvs-bulk-ingest-instructions.md

```diff
@@ -0,0 +1,76 @@
+# Bulk Ingest Instructions
```

Comment:
I'd prefer that these instructions are integrated into the main beta docs instead of a new separate document.

Reply:
that feels like a new ticket though--specifically adding this to beta documentation

Reply:
I was planning to create a new ticket for taking these instructions and using them to update our various "docs with directions."

Reply:
Got it - in that case can we just move this doc out of the beta_docs folder in the meantime? Thanks!